src/components/PluginPage/locales/en-US.ts

```diff
@@ -1,163 +1,172 @@
 export default {
-  'PluginForm.plugin.limit-conn.desc': 'Limit the number of concurrent connections',
-  'PluginForm.plugin.limit-conn.property.conn': 'conn',
-  'PluginForm.plugin.limit-conn.property.conn.extra': 'Maximum number of concurrent connections',
-  'PluginForm.plugin.limit-conn.property.burst': 'burst',
-  'PluginForm.plugin.limit-conn.property.burst.extra':
+  'pluginpage.plugin.limit-conn.desc': 'Limit the number of concurrent connections',
+  'pluginpage.plugin.limit-conn.property.conn': 'conn',
+  'pluginpage.plugin.limit-conn.property.conn.extra': 'Maximum number of concurrent connections',
```

Comment:
Please use PluginPage instead.

Reply:
Can I use lower case ?

Reply:
Better not, we may obey the `ModuleName.key1.key2` style.

Reply:
The demo in umijs maybe all lower case.

Reply:
pluginpage is grouped by plugin with page, we need some ways to let developers know clearly what it is.

Reply:
OK

Reply:
I have changed "pluginpage" to "PluginPage".